src/terminal/Terminal.gd

```diff
@@ -0,0 +1,42 @@
+class_name Terminal
+extends Node
+
+@export var start_window : BaseTerminalWindow;
+
+@onready  var content : MarginContainer = $ContentContainer;
+
+var window_stack : Array;
+
+func _ready():
+	add_window(start_window);
+
```

Comment:
this automatically adds a window when the terminal node is added to the tree. i'm not sure if there is a way for a wrapping scene (such as a Computer scene which would use a Terminal internally) to configure what is displayed. add_window should be made private (`_add_window`) and a `set_window` should be added instead of this `_ready`.
The example scene (or anything at a greater nesting level) can then call upon this node and tell it to display whatever *it* wants to.